components/expression/__examples__/example.jsx

```diff
@@ -0,0 +1,498 @@
+		} else {
+			conditions.splice(i, 1);
+			this.setState({ conditions });
+		}
+	}
+
+	addGroup() {
+		if (!this.props.isChild) {
+			const { conditions } = this.state;
+			const newCondition = {
+				resource: '',
+				operator: '',
+				value: '',
+			};
+			const newGroup = {
+				isGroup: true,
+				triggerType: 'all',
+				conditions: [newCondition],
+			};
+			conditions.push(newGroup);
+			this.setState({ conditions });
+		}
+	}
+
+	updateGroupData(i, val, type) {
+		const { conditions } = this.state;
+		conditions[i][type] = val;
+		this.setState({ conditions });
+	}
+
+	render() {
+		return (
+			<IconSettings iconPath="/assets/icons">
+				<Expression
+					id="expression-example"
+					events={{
+						onChangeTrigger: (val) => {
+							log({
+								action: this.props.action,
+								event,
+								eventName: 'Trigger Changed',
+								data: val,
+							});
+							this.setState({ triggerType: val });
+						},
+						onChangeCustomLogicValue: (event, data) => {
+							log({
+								action: this.props.action,
+								event,
+								eventName: 'Custom Logic Changed',
+								data,
+							});
+							this.setState({ customLogic: data.value });
+						},
+						onAddCondition: () => {
+							log({
+								action: this.props.action,
+								event,
+								eventName: 'New Condition Added',
+								data: null,
+							});
+							this.addCondition();
+						},
+						onAddGroup: () => {
+							log({
+								action: this.props.action,
+								event,
+								eventName: 'New Group Added',
+								data: null,
+							});
+							this.addGroup();
+						},
+					}}
+					triggerType={this.state.triggerType}
+					customLogicValue={this.state.customLogic}
+				>
+					{this.state.triggerType === 'formula' ? (
+						<ExpressionFormula
+							id="expression-formula"
+							resourceCombobox={
+								<Combobox
+									labels={{
+										placeholder: 'Insert a Resource',
+									}}
+									id="expression-formula-insert-resource"
+									multiple={false}
+									options={ResourcesList}
+									selection={
+										this.state.conditions.resource
+											? [
+													find(ResourcesList, {
+														id: this.state.conditions.resource,
+													}),
+												]
+											: []
+									}
+									events={{
+										onSelect: (event, data) => {
+											this.updateFormula(data.selection[0].id, 'resource');
+											log({
+												action: this.props.action,
+												event,
+												eventName: `Formula Resource Changed`,
+												data,
+											});
+										},
+									}}
+									variant="inline-listbox"
+								/>
+							}
+							events={{
+								onChangeTextEditor: (event, data) =>
+									log({
+										action: this.props.action,
+										event,
+										eventName: `Formula updated in Text Editor`,
+										data,
+									}),
+								onClickCheckSyntax: () =>
+									log({
+										action: this.props.action,
+										event,
```

Comment:
Same here, since there's no event variable here, set this to either an empty object or `null`